### DISCUSSION TITLE
HLS streaming from nginx

### DISCUSSION THREAD

**Discussion by neon29** [RaspAP/raspap-webgui#904] [Q&A]
So far, I have used a Rpi to serve an HLS stream to client(s) when both are connected to my home's internet box. 
To achieve this, I currently rely on `nginx` server to stream HLS to `http://mypi.local/hls/index.m3u8` after adding

```
 location /hls/ {
        # /run/shm corresponds to the location where the stream is written 
        root /run/shm;
    }
```
to `/etc/nginx/sites-available/default`

However, rather than using my home's internet box, I would like my client to be able to directly (still through WiFi) connect to the pi to access the stream, and raspap seemed the perfect tool to reach this goal.

However, how can I access this stream as, after raspap installation, `http://mypi.local` redirects to the raspap web interface?
Or, put another way, how can I run a second (raspap web interface being the first) server dedicated to HLS streaming?

**Comment by billz** [comment]
> how can I run a second (raspap web interface being the first) server dedicated to HLS streaming?

https://docs.raspap.com/faq/#webport
